torch/csrc/jit/codegen/cuda/executor.cpp

```diff
@@ -772,17 +888,25 @@ std::vector<at::Tensor> FusionExecutor::runFusion(
               options_.device,
               c10::nullopt));
         }
+        // Note: aliased output is not returned as output. But we still need it
+        // for kernel execution, so would need to push them to args
         for (const auto& entry : executor_entry->io_alias_indices) {
+          auto tensor_arg_abstract =
+              dynamic_cast<const TensorArgAbstract*>(args[entry.second]);
```

Comment:
NIT: It's fine to use a non-descript name in the loop like `(auto entry : ...)` but it is really nice for readability to name it in the loop. i.e. `auto aliased_input_id = entry.second`